example_code/shmem_put_signal_example.c

```diff
@@ -0,0 +1,49 @@
+#include <stdio.h>
```

Comment:
Formatting of the example code should be more consistent with other example code.

Reply:
Can this be a doc edit and wait till the uniform format is decided from #237

Reply:
@jamesaross any comments on this review comment? can i mark this resolved?

Reply:
marking as resolved as this change can be added as doc edit or made as part of #237. We dont have any standard formating template.